domains/zick.json

```diff
@@ -4,6 +4,11 @@
         "email": "lucasalexandrezick@gmail.com"
     },
     "record": {
-        "A": ["144.22.193.227"]
+        "A": ["144.22.193.227"],
+        "MX": "mx.zoho.com",
+        "MX": "mx2.zoho.com",
+        "MX": "mx3.zoho.com",
+        "TXT": "v=spf1 include:zoho.com ~all",
+        "TXT": "v=DKIM1; k=rsa; p=MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCdZp0j5EFJS+OJzSad09+rt4K73CiPfeNahs2qXKBdaRLSrELeRbh4m8dccFi81fAjtPXpU5EonVVj4fssytg06MJCJWbdb02NqwZGS3tm29maZV2UFBb0Fqj4uqNRy1LKynl37jA8XkshE0xyteyFvX3d9Ibf4n7W1VlDhF1HpwIDAQAB"
```

Comment:
TXT records should be combined into array. Same with MX records. They should be similar to your A records.

Reply:
Thank you! Solved it!